qiskit/pulse/transforms.py

```diff
@@ -236,3 +261,127 @@ def compress_pulses(schedules: List[Schedule]) -> List[Schedule]:
         new_schedules.append(new_schedule)
 
     return new_schedules
+
+
+def _push_append(this: List[interfaces.ScheduleComponent],
+                 other: List[interfaces.ScheduleComponent],
+                 ) -> Schedule:
+    r"""Return a ``this`` with ``other`` inserted at the maximum time over
```

Comment:
```suggestion
    r"""Return ``this`` with ``other`` inserted at the maximum time over
```

Reply:
Addressed in other work and I don't have the specific commit :(